specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2023-07-01/imds.json

```diff
@@ -688,6 +716,18 @@
         "version": {
           "type": "string",
           "description": "This is the version of the platform or marketplace image."
+        },
+        "communityGalleryImageId": {
```

Comment:
How are these IDs consumed? I don't see anything in the API that would use them. Are they used externally? Does the customer need to know more about these resources IDs e.g., are they URIs to some blob storage? Is auth needed in that case or is a SAS URI provided?

Reply:
Normally, most of the fields are consumed from other components on the host node. These fields are exposed for community and direct shared image gallery feature, a part of Azure Compute Gallery. Customers don't need to know more about the resources IDs, they are the IDs of image only when using community or direct shared images.